src/libYARP_sig/include/yarp/sig/ImageFile.h

```diff
@@ -29,7 +29,8 @@ namespace yarp {
                     FORMAT_PGM,
                     FORMAT_PPM,
                     FORMAT_JPG,
-                    FORMAT_NUMERIC
+                    FORMAT_NUMERIC,
+                    FORMAT_PNG
                 };
```

Comment:
Is it missing ` bool YARP_sig_API write(const ImageOf<PixelRgb>& src, const std::string& dest, image_fileformat format = FORMAT_PNG);` ?

Reply:
I believe this is correct, it is the comment `// PPM/PGM format` next to these methods that is misleading...